mandatory/1-fix-functions.js

```diff
@@ -21,7 +21,7 @@ function getMood() {
 }
 
 function greaterThan10(num) {
-  let isBigEnough;
+  let isBigEnough = Number > 10;
```

Comment:
the idea is good, however care must be taken to make sure variable names that mean the same value are named exactly the same throughout

Reply:
true, fixed now  👍